src/commands/createFunction.ts

```diff
@@ -142,7 +142,7 @@ export async function createFunction(
     const template: Template = (await ui.showQuickPick<Template>(templatePicks, templatePlaceHolder)).data;
     telemetryProperties.templateId = template.id;
 
-    if (template.bindingType !== 'httpTrigger') {
+    if (template.functionConfig.isHttpTrigger) {
```

Comment:
The logic has flipped here.  Is that intentional?

Reply:
Noooooopeeeee not intentional :)